app/models/submission.rb

```diff
@@ -122,6 +122,22 @@ def survey_id
     survey&.id
   end
 
+  def next
+    consultation.submissions.find_by("id > ?", id)
+  end
+
+  def prev
+    consultation.submissions.find_by("id < ?", id)
+  end
+
+  def first
+    consultation.submissions.first
```

Comment:
These are vulnerable to the default scope changing the ordering - I'd make it explicit. Also, we probably only want active (not archived) submissions

```suggestion
    consultation.submissions.active.order(:created_at).first
```